<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainWindow</class>
 <widget class="QMainWindow" name="mainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Проверь свою память 2.0</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color:rgb(237, 234, 255)</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>330</y>
      <width>791</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 11pt &quot;MS Shell Dlg 2&quot;;
</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;Выберите произведения, которые были написаны Чеховым в 19 веке и в период 1800-1835(Не пушкиным)&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>10</y>
      <width>681</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Запомните данную талицу, чтобы ответить на сложный вопрос, время не ограничено&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_17">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>403</y>
      <width>141</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Евгений Онегин</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_14">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>374</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Горе от ума</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_15">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>403</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Студент</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_10">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>432</y>
      <width>110</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Дубровский</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_13">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>374</y>
      <width>163</width>
      <height>23</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Капитанская дочка</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_16">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>432</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Облако в штанах</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_11">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>490</y>
      <width>124</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Пиковая дама</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_18">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>461</y>
      <width>203</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Сказка о Золотой рыбке</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_12">
    <property name="geometry">
     <rect>
      <x>369</x>
      <y>461</y>
      <width>255</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Послушайте!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>520</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Готово</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>300</y>
      <width>231</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Начать игру</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>490</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Каштанка</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_2">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>380</y>
      <width>131</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Вишневый сад</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_3">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>410</y>
      <width>111</width>
      <height>17</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Палата №6</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>550</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(255, 255, 255);
font: 12pt &quot;MS Shell Dlg 2&quot;;</string>
    </property>
    <property name="text">
     <string>Выйти</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="testnew.qrc"/>
 </resources>
 <connections/>
</ui>
